index.php

```diff
@@ -186,13 +186,21 @@ function validateEmail($email) {
 			&& preg_match('/@.+\./', $email);
 }
 
-// Validates wheter the uploaded file is of
+function validateTickets($files) {
+	$success = true;
+	for($i = 0; $i < count($files['name']); $i++) {
```

Comment:
don't you want to count `$files` here, instead of the length of the `name` attribute?

Reply:
No, that would count the attributes, remember `$files` is SoA here.
See the next comment about the consideration on changing this.